state-chain/pallets/cf-environment/src/migrations/oracle_query_environment.rs

```diff
@@ -0,0 +1,40 @@
+use crate::*;
+
+use frame_support::{pallet_prelude::Weight, traits::UncheckedOnRuntimeUpgrade};
+
+use cf_chains::evm::H256;
+pub struct OracleQueryEnvironmentMigration<T>(PhantomData<T>);
+
+impl<T: Config<Hash = H256>> UncheckedOnRuntimeUpgrade for OracleQueryEnvironmentMigration<T> {
+	fn on_runtime_upgrade() -> Weight {
+		log::info!("🌮 Running migration for Environment pallet: Updating AddressCheckerAddress.");
```

Comment:
I always put a log::info at the end of the runtime upgrade as well.

Also it would be nice to add a pre- and post migration check to ensure the hash has changed (on non-local net of course) just in case